data/odt/styles.xml

```diff
@@ -24,7 +24,7 @@ xmlns:of="urn:oasis:names:tc:opendocument:xmlns:of:1.2"
 xmlns:xhtml="http://www.w3.org/1999/xhtml"
 xmlns:grddl="http://www.w3.org/2003/g/data-view#"
 xmlns:officeooo="http://openoffice.org/2009/office"
-xmlns:tableooo="http://openoffice.org/2009/table"
+xmlns:tableooo="http://openoffice.org/2009/
```

Comment:
Why tihs change? It seems unrelated.

Reply:
That was a mistake, I didn't intend that edit... Reverted